src/platform/testing/e2e/cypress/plugins/index.js

```diff
@@ -12,56 +11,101 @@ const tableConfig = {
 };
 
 module.exports = async on => {
-  const ENV = 'localhost';
-  const publicPath = '/generated/';
-  let outputOptions = {};
+  let appRegistry;
+  if (process.env.CYPRESS_CI) {
+    // eslint-disable-next-line import/no-unresolved
+    appRegistry = require('../../../../../../content-build/src/applications/registry.json');
+  } else {
+    appRegistry = require('../../../../../../../content-build/src/applications/registry.json');
+  }
+  // eslint-disable-next-line no-useless-escape
+  const nodeModules = new RegExp(/^(?:.*[\\\/])?node_modules(?:[\\\/].*)?$/);
+  const dirnamePlugin = {
+    name: 'dirname',
 
-  // Import our own Webpack config.
-  await require('../../../../../../config/webpack.config.js')(ENV).then(
-    webpackConfig => {
-      // Get the original DefinePlugin so we can use __REGISTRY__
-      const definePlugin = webpackConfig.plugins.find(
-        plugin => plugin.constructor.name === 'DefinePlugin',
-      );
+    setup(build) {
+      // eslint-disable-next-line consistent-return
+      build.onLoad({ filter: /.js?$/ }, ({ path: filePath }) => {
+        if (!filePath.match(nodeModules)) {
```

Comment:
If you're trying to detect if `node_modules` is a directory in the path, you can just do:

```suggestion
        if (!filePath.split(path.sep).includes('node_modules')) {
```